<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTeamDialog</class>
 <widget class="QDialog" name="NewTeamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>450</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeamName">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTeamID">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelTeamName">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>140</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>球队名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTeamID">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>球队编号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddPlayer">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>新增球员</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ShowPlayerstableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>队员编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>队员名字</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>队员年龄</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewTeamDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewTeamDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
